platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/MapStrictModeException.java

```diff
@@ -1,6 +1,6 @@
 package com.mapbox.mapboxsdk;
 
-class MapStrictModeException extends RuntimeException {
+public class MapStrictModeException extends RuntimeException {
```

Comment:
Out of curiosity, why was this needed?

Reply:
It was referred from JavaDoc in MapView.java and was erroring.
But also it is thrown in response to a public API call, and being internal does not make sense to me.
![Screen Shot 2022-12-14 at 9 47 10 AM](https://user-images.githubusercontent.com/180515/207669517-f3dd78c1-00fe-410f-80b1-fcd6a5a7a9e7.png)

Reply:
Ok got it, then it's fine :)